### ISSUE TITLE
Vim motion is inconsistent between normal and insert mode

### ISSUE THREAD

**Issue by ndtoan96** [silverbulletmd/silverbullet#948]
In normal mode, going up and down actually jump between paragraphs. While in insert mode, going up and down actually jump between lines.

https://github.com/user-attachments/assets/b88be0ad-3128-4aca-a79c-43895941ecda

**Comment by meain** [comment]
One quick hack to have visual line based navigation would be add a [VIMRC](https://silverbullet.md/VIMRC) and add the following.

```
nnoremap j gj
nnoremap k gk
```

**Comment by ndtoan96** [comment]
> One quick hack to have visual line based navigation would be add a [VIMRC](https://silverbullet.md/VIMRC) and add the following.
> 
> ```
> nnoremap j gj
> nnoremap k gk
> ```

Yes, I did that already. But considering that silverbullet mostly deals with soft-wrap text and don't need line number consistency, I think we can safely set this as default.

**Comment by meain** [comment]
Vim's default is not visual line navigation and I think the editor is emulating that.
